src/main/resources/com/sonyericsson/jenkins/plugins/bfa/db/MongoDBKnowledgeBase/config.jelly

```diff
@@ -39,10 +39,10 @@
         <f:password field="password" />
     </f:entry>
     <f:entry title="${%Enable statistics logging}">
-        <f:checkbox field="enableStatistics" default="true" />
+        <f:checkbox field="enableStatistics" default="true"/>
```

Comment:
I would revert the whitespace changes in this file, they're distracting too the reviewer...

Reply:
Turn on "Hide whitespace changes" ;)

Reply:
Instead of adding another commit where I change the whitespace back, I'll just leave it as it is,
it's not like the whitespace in this file makes sense anyway.